packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Routes/AddRuleDrawer.tsx

```diff
@@ -0,0 +1,365 @@
+            <Box>
+              <LinkButton onClick={onAddServiceTarget}>
+                Add Service Target
+              </LinkButton>
+            </Box>
+          </Stack>
+          <Stack spacing={1.5}>
+            <Typography variant="h3">Session Stickiness</Typography>
+            <Typography>
+              Controls how subsequent requests from the same client are routed
+              when when selecting a backend target. When disabled, no session
+              information is saved.
+            </Typography>
+            {stickinessGeneralError && (
+              <Notice
+                spacingBottom={0}
+                spacingTop={12}
+                text={stickinessGeneralError}
+                variant="error"
+              />
+            )}
+            <FormControlLabel
+              control={
+                <Toggle
+                  checked={isStickynessEnabled}
+                  onChange={onStickinessChange}
+                  sx={{ marginLeft: -1.5 }}
+                />
+              }
+              label="Use Session Stickiness"
+            />
+            {isStickynessEnabled && (
+              <>
+                <Autocomplete
+                  onChange={(_, option) => {
+                    formik.setFieldValue(
+                      'match_condition.session_stickiness_ttl',
+                      option?.label === 'Load Balancer Generated' ? 8 : null
+                    );
+                    formik.setFieldValue(
+                      'match_condition.session_stickiness_cookie',
+                      option?.label === 'Load Balancer Generated' ? null : ''
+                    );
+                  }}
+                  disableClearable
+                  label="Cookie type"
+                  options={stickyOptions}
+                  textFieldProps={{ noMarginTop: true }}
+                  value={cookieType}
+                />
+                {cookieType.label === 'Origin' && (
+                  <TextField
+                    errorText={getMatchConditionError(
+                      'session_stickiness_cookie'
+                    )}
+                    value={
+                      formik.values.match_condition.session_stickiness_cookie
+                    }
+                    label="Cookie"
+                    name="match_condition.session_stickiness_cookie"
+                    noMarginTop
+                    onChange={formik.handleChange}
+                  />
+                )}
+                {cookieType.label === 'Load Balancer Generated' && (
+                  <Stack alignItems="flex-end" direction="row" spacing={1}>
+                    <TextField
+                      errorText={getMatchConditionError(
+                        'session_stickiness_ttl'
+                      )}
+                      value={
+                        formik.values.match_condition.session_stickiness_ttl
+                      }
+                      label="Stickyness TTL"
```

Comment:
```suggestion
                      label="Stickiness TTL"
```
Goodbye "stickyness" 👋🏼